clang/test/CodeGenSYCL/pointers.cpp

```diff
@@ -0,0 +1,36 @@
+// RUN: %clang_cc1 -fsycl -fsycl-is-device -fsycl-int-header=%t.h %s
+// RUN: FileCheck -input-file=%t.h %s
+
+//==--pointer-int-header.cpp - USM kernel param aspace and int header test --==//
```

Comment:
> This one https://github.com/intel/llvm/blob/sycl/clang/test/CodeGenSYCL/pointers-in-structs.cpp will likely fail after your change.

Yes it did. I modified the test in this commit.

Reply:
I meant this test to only check integration header because address spaces are already checked in pointers-in struct.cpp. Thinking about it now,  I should just combine the tests.

Reply:
I kept the 2 tests separate because I noticed that the IR and integration header was checked in different tests for arrays. It keeps things simple in the test as well. I renamed pointers.cpp to pointer-int-header.cpp and added test description. Required Sema and CodeGen checks were added to existing tests - pointers-int-structs.cpp (checks address space) and built-in-kernel-arg.cpp (checks AST).